MapboxNavigation/RouteMapViewController.swift

```diff
@@ -395,124 +320,148 @@ class RouteMapViewController: UIViewController {
             self.currentStepIndexMapped = routeProgress.currentLegProgress.stepIndex
         }
 
-        if self.annotatesSpokenInstructions {
-            self.mapView.showVoiceInstructionsOnMap(route: self.routeController.routeProgress.route)
+        if self.annotatesSpokenInstructions, let routeController {
```

Comment:
If `notifyDidChange(routeProgress: RouteProgress` should only ever be called when routeControler, how about a guard at the beginning of the method to make that clear.